docs/_docs/reference/other-new-features/indentation.md

```diff
@@ -261,7 +261,8 @@ Indentation can be mixed freely with braces `{...}`, as well as brackets `[...]`
 For instance, consider:
 ```scala
 {
-  val x = f(x: Int, y =>
+  val x = 4
+  f(x: Int, y =>
```

Comment:
It works because for example `val y = x: Int` is valid, it widens the type of the expression to `Int`

And since you can use arbitrary expressions as parameters, `val y = f(x: Int)` is valid

Reply:
I thought it was some way to illustrate that the parser isn't treating the argument list like 2 lambda parameters? but its strange to use it there. It probably shouldn't be there unless it specifically illustrates something

Reply:
I 100% agree, I didn't change it because I wanted this PR to be minimal

I did not look at other parts of this page than the examples either, there might be other improvements possible